src/ypd/servlets/user_servlet.py

```diff
@@ -1,27 +1,105 @@
-from flask import flash, redirect, render_template, request, url_for
-from flask_classy import FlaskView, route
-from flask_login import login_required, login_user, logout_user
+import os
+from datetime import datetime, timedelta
+
+from flask import (Markup, current_app, flash, redirect, render_template, request, url_for)
+from flask_login import current_user, login_required, login_user, logout_user
 from sqlalchemy.exc import IntegrityError
 
-from ypd.form.user_form import LoginForm, RegistrationForm
+from flask_classy import FlaskView, route
+from flask_mail import Mail, Message
+from ypd.form.user_form import (ChangePasswordForm, LoginForm, RegistrationForm, ValidateUserForm)
 from ypd.model.user import User, UserType
 
+# from ..server import mail
+
 """A class that represents User creation routes"""
 class UserView(FlaskView):
+    msg = ""
     # Routes work
     @route('/login/', methods=['POST', 'GET'])
     def login(self):
         form = LoginForm()
-        if request.method == 'POST' and form.validate_on_submit:
+        if form.validate_on_submit:
```

Comment:
We need the `request.method == 'POST'` portion to prevent getting error messages before we try to submit the form